src/relay/backend/compile_engine.cc

```diff
@@ -82,10 +82,13 @@ class ScheduleGetter :
     cache_node->func_name = readable_name_stream_.str();
     CachedFunc cfunc(cache_node);
     CHECK(master_op_.defined());
-    Schedule schedule = fschedule[master_op_](
-        master_attrs_, cache_node->outputs, target_);
-    for (const auto& scalar : scalars_) {
-      schedule[scalar].compute_inline();
+    Schedule schedule;
+    if (cache_node->func_name != "__copy") {
```

Comment:
Is there a better way to implement copy? @tqchen thoughts?

Reply:
device_copy has already registered an attribute. We can probably check that.